docs/slack-connector/README.md

```diff
@@ -0,0 +1,33 @@
+# Slack Connector Installation<!-- omit in toc -->
+
+- [Overview](#overview)
+- [Installation in Kyma with Helm](#installation-in-kyma-with-helm)
+  - [Prerequisites](#prerequisites)
+  - [Steps](#steps)
+
+## Overview
+
+Slack Connector is a component which allows contact from inside of Kyma environment to the Slack API.
+
+## Installation in Kyma with Helm
+
+### Prerequisites
+
+- Connection to Kyma cluster
+- Slack Connector Docker image
+
+### Steps
+
+1. Visit our [authentication page](https://auth-slack.herokuapp.com/). Click `Add to Slack`. You will be redirected to another page. Select your desired workspace and click `Allow`
+    >**NOTE:** If the link doesn't work it means that you have to create an application like that yourself. The tutorial for that can be found in the Slack API [documentation](https://api.slack.com/docs/oauth#flow).
```

Comment:
```suggestion
    >**NOTE:** If the link does not work, it means that you have to create an application like that yourself. To create an application, see [this tutorial](https://api.slack.com/docs/oauth#flow) in the Slack API documentation.
```
- Don't use contractions, use full words (_do not_ instead of _don't_). 
- I'm not sure what you mean by 
> ... an application like that

I think you need to be more specific. For example:
> If the link does not work, it means that an application that [does something - specify what it is it does] does not exist and you have to create it yourself. 

or 

>  If the link does not work, it means that there is no application that [does something - specify what it is it does].

and then:
> To create an application [that does X], see ...